cmd/ferretdb/ping.go

```diff
@@ -66,38 +72,8 @@ func ping() {
 	}
 
 	if cli.Listen.TLS != "" {
-		host, port, err := net.SplitHostPort(cli.Listen.TLS)
-		if err != nil {
-			logger.Fatal("Getting host and port failed.", zap.Error(err))
-		}
-
-		l.Debugf("--listen-tls flag is set. Ping to %s will be performed.", cli.Listen.Addr)
-
-		if host == "" {
-			host = "127.0.0.1"
-
-			l.Debugf("Host not specified, defaulting to %s.", host)
-		}
-
-		if cli.Listen.TLSKeyFile == "" || cli.Listen.TLSCaFile == "" {
-			logger.Fatal("When --listen-tls is set, both --listen-tls-cert-file and --listen-tls-ca-file need to be provided.")
-		}
-
-		values := url.Values{}
-
-		values.Add("tls", "true")
-		values.Add("tlsCaFile", cli.Listen.TLSCaFile)
-		values.Add("tlsCertificateKeyFile", cli.Listen.TLSKeyFile)
-
-		u := &url.URL{
-			Scheme:   "mongodb",
-			Host:     net.JoinHostPort(host, port),
-			Path:     cli.Setup.Database,
-			User:     url.UserPassword(cli.Setup.Username, cli.Setup.Password),
-			RawQuery: values.Encode(),
-		}
-
-		urls = append(urls, u.String())
+		// TODO https://github.com/FerretDB/FerretDB/issues/4427
+		l.Warn("TLS ping is skipped, please check https://github.com/FerretDB/FerretDB/issues/4427 for more details.")
```

Comment:
I don't understand why it doesn't work after reading the issue :thinking:

Reply:
I left it here, with motivation to speed up the process of finishing the current issue. I wanted to work on this task right afterwards, and just quickly skip the code that makes TLS healthchecks unusable. I'll improve the issue description

Reply:
```
l.Warn("TLS ping is not implemented yet.")
```

TODO with an issue number is enough.

Reply:
Yep good idea to scope out tricky things :+1: